.github/workflows/main.yml

```diff
@@ -57,6 +57,27 @@ jobs:
         name: wasi_snapshot_preview1.command.wasm
         path: wasi_snapshot_preview1.command.wasm
     - run: cargo build --target wasm32-unknown-unknown --release
+
+    - run: cargo install --git https://github.com/bytecodealliance/wit-bindgen wit-bindgen-cli
```

Comment:
I'd recommend using `--rev` here as well to avoid this breaking as `wit-bindgen` changes over time